classes/controller/FrontController.php

```diff
@@ -1410,7 +1410,7 @@ public function getTemplateFile($template, $params = [], $locale = null)
         }
 
         if ($overridden_template = Hook::exec(
-            'DisplayOverrideTemplate',
+            'displayOverrideTemplate',
```

Comment:
Hello @Amit-Kumar-Tiwari-Webkul are you referring to issue https://github.com/PrestaShop/PrestaShop/issues/27854 ?

Reply:
@matks , Yes I am referring to this issue #27854

Reply:
https://github.com/PrestaShop/PrestaShop/issues/27854 has been fixed in PrestaShop 1.7.8.5 😄  

So now we must ask ourselves if this change is desired. Should we put this hook call in lowercase as you suggest or should we keep it as it was before?

ping @kpodemski wdyt

Reply:
@matks it's not required, but it might be suitable for a consistency